arangod/RestHandler/RestReplicationHandler.cpp

```diff
@@ -2058,7 +2058,7 @@ void RestReplicationHandler::handleCommandSync() {
   // will throw if invalid
   config.validate();
 
-  TRI_ASSERT(!config._skipCreateDrop);
+  //TRI_ASSERT(!config._skipCreateDrop);
```

Comment:
if its not needed anymore pls remove, else state in a comment why its still here.